src/components/users/viewCohorts/viewCohorts.jsx

```diff
@@ -1,10 +1,45 @@
-import React from "react"
-import { useParams } from "react-router-dom"
-import "./viewCohort.css"
-import Header from "../../Header/Header.jsx"
+import React from "react";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import "./viewCohort.css";
+import Header from "../../Header/Header.jsx";
+import client from "../../../utils/client";
 
 export default function ViewCohort() {
-  const { id } = useParams()
+  const { id } = useParams();
+  const [noCohort, setNoCohort] = useState([]);
+
+  useEffect(() => {
+    client
+      .get(`/user`)
+      .then((res) => {
+        const noCo = res.data.data.users.filter(
+          (user) => user.role === "STUDENT" && user.cohort_id === null
```

Comment:
I would suggest moving this filtering logic to the server - make a new server endpoint that returns students that are not currently part of a chorot. The reason being, if you had 100's (or 1000's)of students, but only a handful not in a Cohort, you'd have to load all of them at once just to display only a few. It also means that if you change the definition of what "not in a cohort" means, you only need to update the server endpoint and not the client. Right now, cohort_id being null means a student is not in a cohort - but that's really an implementation detail the server should take care off.  Does that make sense?

Reply:
We can't implement this until the backend is done. Should we close the PR and re open when this can be sorted or just wait until that point and leave it open?

Reply:
I think it's fine to leave it open for now.